scipy/linalg/decomp_svd.py

```diff
@@ -254,6 +257,7 @@ def diagsvd(s, M, N):
 
     Examples
     --------
+    >>> import numpy as np
```

Comment:
This line is implicit to all examples in SciPy hence we omit it.

Reply:
Ok thanks I will fix all of these.